database/frontend/react.json

```diff
@@ -1,10 +1,17 @@
 [
     {
-        "name":"React js",
-        "description":"Free online course for learning react js",
-        "url":"https://scrimba.com/learn/learnreact#",
-        "category":"frontend",
-        "subcategory":"react",
-        "language":"english"
+        "name": "React js",
+        "description": "Free online course for learning react js",
+        "url": "https://scrimba.com/learn/learnreact#",
+        "category": "frontend",
+        "subcategory": "react",
+        "language": "english"

```

Comment:
```suggestion
```

Reply:
You can see the file here: https://github.com/rupali-codes/LinksHub/blob/aea64406f8f4a21f2478650ec12d7166699ff5d3/database/frontend/react.json

Apologies for the confusion. Despite not being the original contributor, I have made the necessary changes as required.